docs/user-guide/release-notes/2.21.x.rst

```diff
@@ -2,6 +2,16 @@
 Pulp 2.21 Release Notes
 =======================
 
+Pulp 2.21.4
+===========
+
+Bug Fixes
+---------
+
+* Fixed several permissions-issues that blocked pulp2to3 migrations.
```

Comment:
Nitpick. Migrations were not blocked but were requiring much more storage space. So maybe s/blocked/affected/?

`permissions-issues` - not sure I followed this one. Did you mean `permission issues`? or you are calling the thing which was fixed a `permission-issue`?

Reply:
RE nitpick - bah. Well, I guess it stays the way it is; will try to be more exact next time.

RE permissions-issues - The latter - we fixed one permissions-issue ("the permissions on directories were set incorrectly by pulp2"), and then we fixed a second ("the permissions on files were set incorrectly by pulp2"), so now we have two permissions-issues :)